packages/web-components/src/components/dotcom-shell/__stories__/dotcom-shell.stories.ts

```diff
@@ -109,21 +108,23 @@ export default {
   parameters: {
     ...readme.parameters,
     knobs: {
-      'dds-masthead-container': ({ groupId }) => ({
+      Masthead: ({ groupId }) => ({
```

Comment:
Just curious: Is it your preference of knob namespacing?

Reply:
Yes, namely for labeling the tabs in the knob panel so it's clearer to the adopters.